modules/ROOT/pages/apikit-4-prevent-fault-task.adoc

```diff
@@ -1,13 +1,13 @@
-= To Prevent the SOAP Fault
+= Prevent the SOAP Fault
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this procedure, you resolve the SOAP fault you generated. You add the Transform Message component to the flow and specify a message in DataWeave for responding to the request, effectively implementing the operation. In the DataWeave code, you use the proper namespace to avoid a faultcode at the beginning of the response to the request.
+After resolving the xref:apikit-4-soap-fault-task.adoc[SOAP fault] generated, add the Transform Message component to the flow and specify a message in DataWeave for responding to the request and implementing the operation. In the DataWeave code, use the proper `namespace` to avoid a fault code at the beginning of the response to the request.
```

Comment:
```suggestion
After resolving the xref:apikit-4-soap-fault-task.adoc[SOAP fault] generated, add the Transform Message component to the flow and specify a message in DataWeave to respond to the request and implement the operation. In the DataWeave code, use the proper `namespace` to avoid a fault code at the beginning of the response to the request.
```

Reply:
OK! Thanks!